.github/workflows/docker.yml

```
@@ -0,0 +1,16 @@
+name: Build CLI Docker image
+
+on:
+  push:
+    paths:
+      - './Dockerfile'
+
+jobs:
+  verify_image:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      - name: Build Image
+        run: docker build -t atlas .
```

```suggestion
        run: docker build --no-cache -t atlas .
```

Context: When we want to verify image but do not push it we should ignore cache to make sure that every layer works correctly with each other. In production scenario we always want cache as this affects end users.

---

Just FYI We rely here on the docker available in the actions platform (which is ok)
We should likely use github action with a controllable version of the container engine for production builds.